test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -417,7 +417,7 @@ $ kafka-verify format=json sink=materialize.public.json_avro_upsert_key_2 key=tr
 
 # TODO: we've disabled this after observing it be flaky in CI. Re-enable once we've
 # fixed the issue.
-#$ verify-timestamp-compaction source=input_csv max-size=3 permit-progress=true
-$ verify-timestamp-compaction source=rt_binding_consistency_test_source max-size=3 permit-progress=true
-$ verify-timestamp-compaction source=input_kafka_dbz max-size=3 permit-progress=true
-$ verify-timestamp-compaction source=input_kafka_cdcv2 max-size=3 permit-progress=true
+# $ verify-timestamp-compaction source=input_csv max-size=3 permit-progress=true
```

Comment:
Are these tests flaky or broken after this PR?

Reply:
They used to be broken (at some point during development) but let me see if they work now